pages/dev.js

```diff
@@ -18,21 +18,107 @@ export default function Home() {
           email="amanda.dupell@hpe.com"
           linkedIn="https://www.linkedin.com/in/amanda-dupell-profile/"
         />
-        <Paragraph size="large">
-          You are expecting an exciting internship diving into the open-source
-          world and the community of grommet.io, so far we have more than +3,700
-          members on our slack channel, a great team, loads of enhancements
-          requests and an exciting road-map for both grommet and the HPE
-          Design-System.
+        <Text size="4xl" weight="bold" >Hi, there!</Text>
+        <Paragraph size="large">My name is Amanda Dupell and I am super excited to start
+          working as a UI/UX Developer Intern as a part of HPE's Experience 
+          Studio and Design System team!
         </Paragraph>
-        <Paragraph size="large">
-          Shimrit (Shimi) Yacobi, will be your “HPE Onboarding Buddy” (located
-          in Fort Collins, CO) and a primary contact to get you oriented to HPE
-          which includes getting your Mac set up, getting connected to the HPE
-          Network, and addressing any questions about the team and internship
-          project and such.
+        <Paragraph size="medium">
+          As I begin my internship at HPE, I have a lot of exciting projects and 
+          learning opportunities to look forward to this summer and beyond. One of
+          the first things I will be working on is the continuous improvement of 
+          accessibility across {
+            <Anchor href="https://github.com/grommet/grommet">Grommet</Anchor>
+          } components. This process begins by researching and diving head first 
+          into development tools to real-time inspect the web pages and 
+          components I will be contributing to and building. 
         </Paragraph>
-        <Box></Box>
+        <Box pad={{vertical: "large"}}>
+          <Card width="large" elevation="xlarge">
```

Comment:
Hard coding values like `width` might affect the responsive behavior of the application since it doesn't scale.

Reply:
when looking around and playing with the window resizing and mobile inspect tool, i was able to see the card resizing based on the window size. it seems that the width is a max-width and adjusts when the window gets smaller. is this still bad?

https://user-images.githubusercontent.com/38971752/119423222-cd637980-bcd0-11eb-9f61-fde46418488c.mov